python/dataset_plotting.py

```diff
@@ -14,18 +13,41 @@
     print("Warning: the current version of this plotting module was designed to"
           " work inside a Jupyter notebook. Other usage has not been tested.")
 
+#===============================================================================
+
+def check_input(input_data):
+
+    values = []
+    for var in input_data:
+        if var.is_data:
+            values.append(var)
+
+    if len(values) > 1:
+        raise RuntimeError("More than one Data.Value found! Please use e.g."
+                           " plot(dataset.subset('sample'))"
+                           " to select only a single Value.")
+    return values, len(values[0].dimensions)
+
+#===============================================================================
+
 # Wrapper function to dispatch the input dataset to the appropriate plotting
 # function depending on its dimensions
 def plot(input_data):
-    ndim = len(input_data.dimensions)
-    if ndim == 1:
+
+    # A list of datasets is only supported for 1d
+    if type(input_data) is list:
         return plot_1d(input_data)
-    elif ndim == 2:
-        return plot_image(input_data)
-    elif ndim < 5:
-        return plot_sliceviewer(input_data)
+    # Case of a single dataset
     else:
-        raise RuntimeError("Plot: unsupported number of dimensions: {}".format(ndim))
+        values, ndim = check_input(input_data)
```

Comment:
I don't understand. Doesn't this prevent plotting multiple variables or variables with error bars in the 1D case?

Reply:
Hmm yes it does and apparently you do understand ;-) will fix